### ISSUE TITLE
.get include with wildcards doesn't work on localStorage/IndexedDB 

### ISSUE THREAD

**Issue by Bubz43** [appy-one/acebase#115]
When used the node version it works as expected, but with `AceBase.WithLocalStorage` or `AceBase.WithIndexedDB` it doesn't.
```js
// Include specific nested data:
db.ref('users/someuser/posts')
  .get({ include: ['*/title', '*/posted'] })
  .then(snap => {
    // snapshot will be an empty object
});

// Combine include & exclude:
db.ref('users/someuser')
  .get({ exclude: ['comments'], include: ['posts/*/title'] })
  .then(snap => {
    // snapshot will exclude comments but won't contain a posts property at all
});

```

**Comment by appy-one** [comment]
Thanks for reporting! I ran a quick test and was able to reproduce. I'll look into it asap

**Comment by appy-one** [comment]
I've published a fix for this in v1.21.3, let me know if all works as expected!

[![Spread the word](https://user-images.githubusercontent.com/26569719/169265089-3d593555-e1ad-4390-986b-877ac2c38a47.svg)](https://twitter.com/intent/tweet?button=&url=https://github.com/appy-one/acebase&text=I'm+using+@AcebaseRealtime+in+my+project+to+make+my+life+easier!&button=)  [![contribute](https://user-images.githubusercontent.com/26569719/169265318-30c4c6a5-7c89-46a0-a7a2-ef433a8192f4.svg)](https://github.com/appy-one/acebase#contributing)  [![Sponsor AceBase](https://user-images.githubusercontent.com/26569719/168233053-8e56b243-4140-40ab-9a30-4cb3cc149bfe.svg)](https://github.com/sponsors/appy-one)

**Comment by Bubz43** [comment]
Works great, thanks for the quick fix.
